lib/woot_message_seeder.rb

```diff
@@ -0,0 +1,100 @@
+module WootMessageSeeder
```

Comment:
@sojan-official Can you please extract this seeder out and try to close this PR?

Reply:
https://github.com/chatwoot/chatwoot/pull/1676